### DISCUSSION TITLE
Problem with Chrome Plugin

### DISCUSSION THREAD

**Discussion by taminelli** [dani-garcia/vaultwarden#3259] [Q&A] [answered]
### Discussed in https://github.com/dani-garcia/vaultwarden/discussions/3257

<div type='discussions-op-text'>

<sup>Originally posted by **taminelli** February 18, 2023</sup>
I have reinstall my pc. After that I can no longer login with chrome plugin.
URL is correct. [https://bitwarden.domain.com:5152 1](https://bitwarden.domain.com:5152/)

Message: Cannot read properties of null (reading ‘iterations)

Login on website is possible.</div>

**Comment by tessus** [comment]
First of all it would make sense, if you told us the version of the Chrome plugin. If you use the latest one, it will fail, because vw needs to be updated.

But without info, there's nothing the devs can do. The first thing one does in a bug report is to state the versions of the SW involved. You neither told us which vw server version, nor which Chrome plugin version.

**Comment by taminelli** [comment]
Chrome Plugin: 2023.2.0
VW; Version 2.19.0

**Comment by tessus** [comment]
There is no vw server version 2.19.0. And as I mentioned in the discussion thread (next time choose either an issue or a discussion - not both) you have to downgrade the plugin to `2023.1.0`.

**Reply by taminelli** [reply]
after downgrade I have an other message. Passwort or emaiadress incorrect. But its correct. over website I can log in with this data

**Comment by taminelli** [comment]
Sorry I have move from the Diskussion to issue but after that it was twice and not sble to delet. 
![5D37DAB8-98E5-49E0-8C65-2AEC130237BE](https://user-images.githubusercontent.com/115330291/219910020-dacfd4b0-45f6-4607-864f-31af05d923b3.jpeg)

how can I downgrsde plugin?

**Comment by tessus** [comment]
> Sorry I have move from the Diskussion to issue but after that it was twice and not sble to delet.

Ah, I see. The problem is that these 2 threads are out of sync and people have to read both not to miss any information.

How did you install vaultwarden? There is no version 2.19.0. The latest release is 1.27.0 and the web shows the web version not the vw server version:

<img width="316" alt="image" src="https://user-images.githubusercontent.com/223439/219911016-2e31f612-7b23-4572-b5cc-f35a8353dce3.png">


> how can I downgrsde plugin?

In the Chrome web store there is no option to install older versions, but you can download it here: https://github.com/bitwarden/clients/releases/download/browser-v2023.1.0/dist-chrome-2023.1.0.zip

In Firefox one can just download older versions https://addons.mozilla.org/en-US/firefox/addon/bitwarden-password-manager/versions/

**Comment by BlackDex** [comment, marked as answer]
I suggest to install the latest version of Vaultwarden.
The reason that stuff isn't working anymore is because the clients are too new for the server version.

You are on v1.20 of Vaultwarden, almost 2 years old!

So update your Vaultwarden instances first and check again.

**Reply by taminelli** [reply]
Where I can see the server version ? I use Vaultvarden on my synology. I can see only vaultwarden/server:latest

**Reply by BlackDex** [reply]
You could login into the `/admin` interface, which should show you the version. Not sure from the top of my head if your version already has the `/admin/diagnostics` page.

**Reply by taminelli** [reply]
THX! Is there no function to update on the admin interface? Whats de best way to update my wv on syno in docker?

![Screenshot 2023-02-19 132534](https://user-images.githubusercontent.com/115330291/219947806-0c92c836-0946-44ff-868a-1906d4280c24.png)

**Reply by BlackDex** [reply]
I have no clue on how this works on a Synology system.
Normally with dockers you pull the image, remove the container, and re-create it.

**Reply by taminelli** [reply]
how do you run your system?

**Reply by taminelli** [reply]
now I have 1.27.0  ant its working 
:)

**Reply by Miicker** [reply]
> now I have 1.27.0 ant its working :)

I have the same issue on my Synology (docker) how did you perform the update?

**Reply by taminelli** [reply]
Delete Vaultwarden in Docker, Install new Version and copy all old settings to the new one.

**Reply by Miicker** [reply]
> Delete Vaultwarden in Docker, Install new Version and copy all old settings to the new one.

And which image did you choose? bitwardenrs?

**Reply by BlackDex** [reply]
bitwardenrs isn't updated anymore!! Only vaultwarden.
